heal/harvard_downloads.py

```diff
@@ -0,0 +1,268 @@
+"""
+This module includes an external file retriever function intended to be called
+by the external_files_download module in the Gen3-SDK.
+
+The retriever function sends requests to the Syracuse QDR API for downloading studies or files.
```

Comment:
There are many places in this file that is referencing to `Syracuse QDR` or `QDR`, which should be replaced by `Harvard Dataverse`